deployments/helm-chart/README.md

```diff
@@ -214,6 +214,7 @@ Parameter | Description | Default
 `controller.reportIngressStatus.leaderElectionLockName` | Specifies the name of the ConfigMap, within the same namespace as the controller, used as the lock for leader election. controller.reportIngressStatus.enableLeaderElection must be set to true. | Autogenerated
 `controller.reportIngressStatus.annotations` | The annotations of the leader election configmap. | {}
 `controller.pod.annotations` | The annotations of the Ingress Controller pod. | {}
+`controller.pod.labels` | The labels of the Ingress Controller pod. | {}
```

Comment:
This parameter doesn't fully control the labels of the Ingress Controller pod, because the templates always add `app: ... ` label.
Considering that, would any of the below ideas make sense?
- extend the documentation a bit. `The labels of the Ingress Controller pod.` -> `Additional labels of the Ingress Controller pod.`
- rename the parameter and update the doc. So it is similar to `service.extraLabels`

Reply:
I've updated it from `labels` to `extraLabels` and added the word `additional` to the docs to make it more obvious